sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -68,6 +68,18 @@ singleStatement
     : statement SEMICOLON* EOF
     ;
 
+label
+    : multipartIdentifier
+    ;
+
+beginLabel
```

Comment:
any reason not not use `multipartIdentifier` directly in `beginLabel` and `endLabel`?

Reply:
Hm, we can but `multipartIdentifier` directly in `beginLabel` and `endLabel`. This was done for clarity, but I agree it is unnecessary to add another level in parser.